ParkingLot/Models/Car.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+
+namespace ParkingLot.Models
+{
+    public class Car
+    {
+        public Car(string brand)
+        {
+            Brand = brand;
+        }
+
+        public string Brand { get; }
+
+        public override string ToString()
+        {
+            return Brand;
+        }
+    }
+
+    public class CarParked
+    {
+        public CarParked(string ticket, Car car)
+        {
+            Ticket = ticket;
+            Car = car;
+        }
+
+        public Car Car { get; }
+        public string Ticket { get; }
+    }
```

Comment:
1. Car maybe do so much things.
2. A car created another instance when it is parked.